README.md

```diff
@@ -75,7 +75,8 @@ We try our best to maintain backwards compatibility of our plugin with previous
 | v0.59                   | v5.6+ *(RN refactored js bundle loader code)*         |
 | v0.60-v0.61             | v6.0+ *(RN migrated to Autolinking)*                  |
 | v0.62-v0.64             | v6.2+ *(RN removed LiveReload)*                       |
-| v0.65-v0.69             | v7.2+ *(RN updated iPhone-target-version)*            |
+| v0.65-v0.69             | v7.0+ *(RN updated iPhone-target-version)*            |
```

Comment:
Isn't it supposed to be 0.65-v0.70 ?

Reply:
Yes that makes sense. I was fixing that line because the Code push version was wrong there is no 7.2, latest version is 7.1